1.0/docs/start/first-element/step-2.md

```diff
@@ -0,0 +1,255 @@
+---
+layout: default
+type: start
+shortname: Start
+title: "Step 2: Add Local DOM"
+subtitle: Build your first Polymer element
+---
+
+<style>
+.caption + pre,.caption + .prettyprint {
+  margin-top: 0px;
+}
+</style>
+
+Next, you'll create a simple element that displays an icon.
+
+In this step, you'll learn about:
+
+*   Creating a custom element using Polymer.
+*   Working with local DOM.
+
+
+<div class="alert alert-info"><strong>Learn more: local DOM.</strong> Local DOM
```

Comment:
This stuck out to me. From the eyes of a new user it's pretty technical. I'm still easing into things at this point. Maybe this should go down in the local DOM section where it can have a more in context description. As it stands I have a bit of a "?" over my head if I'm totally new to things. Like.. "I heard web components used Shadow DOM, what's the difference between Local DOM?"